src/patternfly/components/ToggleGroup/examples/ToggleGroup.md

```diff
@@ -0,0 +1,158 @@
+  {{/toggle-group-button}}
+{{/toggle-group}}
+
+<br/>
+
+{{#> toggle-group}}
+  {{#> toggle-group-button toggle-group-button--attribute='aria-label="Copy button"'}}
+    <i class="fas fa-copy" aria-hidden="true"/>
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--attribute='aria-label="Undo button"'}}
+    <i class="fas fa-undo" aria-hidden="true"></i>
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--attribute='aria-label="Share button" disabled'}}
+    <i class="fas fa-share-square" aria-hidden="true"></i>
+  {{/toggle-group-button}}
+{{/toggle-group}}
+```
+
+```hbs title=Light
+{{#> toggle-group}}
+  {{#> toggle-group-button toggle-group-button--modifier="pf-m-light"}}
+    Option 1
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--modifier="pf-m-light"}}
+    Option 2
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--modifier="pf-m-light pf-m-selected"}}
+    Option 3
+  {{/toggle-group-button}}
+{{/toggle-group}}
+
+<br/>
+
+{{#> toggle-group}}
+  {{#> toggle-group-button toggle-group-button--modifier="pf-m-light" toggle-group-button--attribute='aria-label="Copy button"'}}
+    <i class="fas fa-copy" aria-hidden="true"/>
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--modifier="pf-m-light" toggle-group-button--attribute='aria-label="Undo button"'}}
+    <i class="fas fa-undo" aria-hidden="true"></i>
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--modifier="pf-m-light pf-m-selected" toggle-group-button--attribute='aria-label="Share button"'}}
+    <i class="fas fa-share-square" aria-hidden="true"></i>
+  {{/toggle-group-button}}
+{{/toggle-group}}
+```
+
+
+## Documentation
+### Overview
+
+
+### Accessibility
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `aria-label="[button label text]"` | `.pf-c-toggle-group__button` | Provides an accessible name for the button when an icon is used instead of text. **Required when icon is used with no supporting text** |
+| `disabled` | `.pf-c-toggle-group__button` | When a button element is used, indicates that it is unavailable and removes it from keyboard focus. **Required when button is disabled** |
+| `aria-disabled="true"` | `button.pf-c-toggle-group__button` | When a button element is used, indicates that it is unavailable but does not prevent keyboard or hover interactions. Used when a disabled button provides interactive elements like a tooltip. |
```

Comment:
why do you need `aria-disabled`? shouldn't be necessary with `disabled`

Reply:
I was thinking that the buttons in the toggle group would be the same as the buttons in the button component - but i guess i dont need it because they have more of a specific use case and wouldnt need an interactive element like a tooltip